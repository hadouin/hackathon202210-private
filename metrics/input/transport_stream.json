{
	
	"arte": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/data/media/test_arte.ts"
		}
	},
	"m6": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/data/media/test_m6.ts"
		}
	},
	"astra": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/data/media/test_astra.ts"
		}
	},
	"natgeo": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/data/media/test_natgeo.ts"
		}
	},
	"luxetv": {
		"type": "parse_transport_stream",
		"arguments": {
			"filename": "/data/media/test_luxetv.ts"
		}
	}
}
