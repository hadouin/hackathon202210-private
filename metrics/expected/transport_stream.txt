arte [{'pid': 3243, 'deltaPcrPts': {'min': 403, 'max': 1103, 'average': 1010}, 'duration': 668, 'size': 861355276, 'bandwidth': 10308841}, {'pid': 3353, 'deltaPcrPts': {'min': 28, 'max': 69, 'average': 49}, 'duration': 668, 'size': 10987472, 'bandwidth': 131521}, {'pid': 3343, 'deltaPcrPts': {'min': 11, 'max': 51, 'average': 31}, 'duration': 668, 'size': 21912528, 'bandwidth': 262296}]
m6 [{'pid': 203, 'deltaPcrPts': {'min': 1058, 'max': 2131, 'average': 1435}, 'duration': 779, 'size': 725414544, 'bandwidth': 7445685}, {'pid': 303, 'deltaPcrPts': {'min': 37, 'max': 102, 'average': 71}, 'duration': 779, 'size': 38323988, 'bandwidth': 393437}]
astra [{'pid': 257, 'deltaPcrPts': {'min': 372, 'max': 1166, 'average': 661}, 'duration': 137, 'size': 176613592, 'bandwidth': 10247379}, {'pid': 256, 'deltaPcrPts': {'min': 51, 'max': 72, 'average': 63}, 'duration': 137, 'size': 4611076, 'bandwidth': 267448}]
natgeo [{'pid': 257, 'deltaPcrPts': {'min': 1177, 'max': 2302, 'average': 1534}, 'duration': 209, 'size': 186044424, 'bandwidth': 7096660}, {'pid': 258, 'deltaPcrPts': {'min': 39, 'max': 74, 'average': 57}, 'duration': 209, 'size': 5257044, 'bandwidth': 200612}]
luxetv [{'pid': 3013, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3014, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3015, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3016, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3017, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3018, 'deltaPcrPts': {'min': 135, 'max': 174, 'average': 155}, 'duration': 267, 'size': 5530396, 'bandwidth': 165318}, {'pid': 3011, 'deltaPcrPts': {'min': 563, 'max': 1496, 'average': 1011}, 'duration': 267, 'size': 192211012, 'bandwidth': 5750516}]
